src/bank.rs

```diff
@@ -974,6 +996,15 @@ mod tests {
         (genesis.into_iter().chain(block), mint.pubkey())
     }
 
+    fn create_sample_ledger_with_mint_and_keypairs(
+        mint: Mint,
```

Comment:
No need to move the Mint here. Borrow it, and then you don't need to return its Pubkey.